kvpress/presses/tova_press.py

```diff
@@ -0,0 +1,46 @@
+# SPDX-FileCopyrightText: Copyright (c) 1993-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+from dataclasses import dataclass
+
+import torch
+from torch import nn
+import torch.nn.functional as F
+
+from kvpress.presses.snapkv_press import SnapKVPress
+
+
+@dataclass
+class TOVAPress(SnapKVPress):
+    """
+    TOVA (https://arxiv.org/abs/2401.06104) use the attention of the last token averaged across heads
```

Comment:
Maybe also link to 
https://github.com/schwartz-lab-NLP/TOVA/blob/main/src/tova_cache.py